workflow_tests/utils/ExtendedAct.ts

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable @typescript-eslint/dot-notation */
+// This eslint-disable comment is here to allow accessing private properties in the Act class
+import type {RunOpts, Workflow} from '@kie/act-js';
+import * as kieActJs from '@kie/act-js';
+import path from 'path';
+import {JobMocker} from './JobMocker';
+import type {MockJob} from './JobMocker';
+
+type ExtendedActOpts = RunOpts & {actor?: string; workflowFile?: string; mockJobs?: Record<string, MockJob>};
```

Comment:
I saw `Record<string, MockJob>` several times through the code, maybe we can put it in the separate type and reuse it in different places?